app/services/force/client_factory.rb

```diff
@@ -1,23 +1,30 @@
 module Force
-  # Salesforce Module
+  # Creates clients for quering saleforce
   class ClientFactory
-    # TODO: Update this with a top-level comment
     def new_for_user(user)
       if Rails.env.test?
-        new_with_username_and_password
+        # prybug
+        # new_with_username_and_password
+        new_with_oauth_token(AccessToken.request_new_with_credentials)
       else
         new_with_oauth_token(user.oauth_token)
       end
     end
 
     def username_and_password_attributes
+      # {
```

Comment:
is this commented out code, or meant to be a helpful docstring? If it's meant to be helpful, can you add some more details